src/org/exist/xquery/functions/fn/FunUnparsedText.java

```diff
@@ -36,7 +38,9 @@
 import java.nio.charset.StandardCharsets;
 
 import static org.exist.xquery.FunctionDSL.*;
+import static java.nio.charset.StandardCharsets.UTF_8;
 
+@RunWith(ParallelRunner.class)
```

Comment:
junit reference in a non-test class? Is this intentional?

Reply:
Oops, that went into the wrong class. I have fixed that now.